29-Aug-2024 16:00:30.408 SEVERE [http-nio-8080-exec-3] org.apache.catalina.core.StandardWrapperValve.invoke Servlet.service() for servlet [com.wavemaker.leavemanagement.controller.LeavesController] in context with path [/leave_management] threw exception
	org.json.JSONException: JSONObject["employeeId"] not found.
		at org.json.JSONObject.get(JSONObject.java:596)
		at org.json.JSONObject.getInt(JSONObject.java:768)
		at com.wavemaker.leavemanagement.service.LeavesService.requestLeave(LeavesService.java:48)
		at com.wavemaker.leavemanagement.controller.LeavesController.doPost(LeavesController.java:65)
		at jakarta.servlet.http.HttpServlet.service(HttpServlet.java:590)
		at jakarta.servlet.http.HttpServlet.service(HttpServlet.java:658)
		at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:195)
		at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:140)
		at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:51)
		at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:164)
		at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:140)
		at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:167)
		at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:90)
		at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:483)
		at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:115)
		at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:93)
		at org.apache.catalina.valves.AbstractAccessLogValve.invoke(AbstractAccessLogValve.java:663)
		at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:74)
		at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:344)
		at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:384)
		at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:63)
		at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:904)
		at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1741)
		at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:52)
		at org.apache.tomcat.util.threads.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1190)
		at org.apache.tomcat.util.threads.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:659)
		at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:63)
		at java.base/java.lang.Thread.run(Thread.java:1583)
29-Aug-2024 16:12:10.372 SEVERE [http-nio-8080-exec-6] org.apache.catalina.core.StandardWrapperValve.invoke Servlet.service() for servlet [com.wavemaker.leavemanagement.controller.LeavesController] in context with path [/leave_management] threw exception
	java.lang.RuntimeException: java.sql.SQLSyntaxErrorException: Column 'EMPLOYEE_ID' not found.
		at com.wavemaker.leavemanagement.repository.impl.EmployeeRepositoryImpl.getEmployeeIdByEmailId(EmployeeRepositoryImpl.java:82)
		at com.wavemaker.leavemanagement.service.LeavesService.getAllLeaves(LeavesService.java:18)
		at com.wavemaker.leavemanagement.controller.LeavesController.doGet(LeavesController.java:32)
		at jakarta.servlet.http.HttpServlet.service(HttpServlet.java:564)
		at jakarta.servlet.http.HttpServlet.service(HttpServlet.java:658)
		at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:195)
		at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:140)
		at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:51)
		at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:164)
		at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:140)
		at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:167)
		at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:90)
		at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:483)
		at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:115)
		at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:93)
		at org.apache.catalina.valves.AbstractAccessLogValve.invoke(AbstractAccessLogValve.java:663)
		at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:74)
		at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:344)
		at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:384)
		at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:63)
		at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:904)
		at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1741)
		at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:52)
		at org.apache.tomcat.util.threads.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1190)
		at org.apache.tomcat.util.threads.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:659)
		at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:63)
		at java.base/java.lang.Thread.run(Thread.java:1583)
	Caused by: java.sql.SQLSyntaxErrorException: Column 'EMPLOYEE_ID' not found.
		at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:112)
		at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:89)
		at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:81)
		at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:55)
		at com.mysql.cj.jdbc.result.ResultSetImpl.findColumn(ResultSetImpl.java:599)
		at com.mysql.cj.jdbc.result.ResultSetImpl.getInt(ResultSetImpl.java:872)
		at com.wavemaker.leavemanagement.repository.impl.EmployeeRepositoryImpl.getEmployeeIdByEmailId(EmployeeRepositoryImpl.java:70)
		... 26 more
29-Aug-2024 16:12:11.698 SEVERE [http-nio-8080-exec-5] org.apache.catalina.core.StandardWrapperValve.invoke Servlet.service() for servlet [com.wavemaker.leavemanagement.controller.LeavesController] in context with path [/leave_management] threw exception
	java.lang.RuntimeException: java.sql.SQLSyntaxErrorException: Column 'EMPLOYEE_ID' not found.
		at com.wavemaker.leavemanagement.repository.impl.EmployeeRepositoryImpl.getEmployeeIdByEmailId(EmployeeRepositoryImpl.java:82)
		at com.wavemaker.leavemanagement.service.LeavesService.getAllLeaves(LeavesService.java:18)
		at com.wavemaker.leavemanagement.controller.LeavesController.doGet(LeavesController.java:32)
		at jakarta.servlet.http.HttpServlet.service(HttpServlet.java:564)
		at jakarta.servlet.http.HttpServlet.service(HttpServlet.java:658)
		at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:195)
		at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:140)
		at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:51)
		at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:164)
		at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:140)
		at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:167)
		at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:90)
		at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:483)
		at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:115)
		at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:93)
		at org.apache.catalina.valves.AbstractAccessLogValve.invoke(AbstractAccessLogValve.java:663)
		at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:74)
		at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:344)
		at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:384)
		at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:63)
		at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:904)
		at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1741)
		at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:52)
		at org.apache.tomcat.util.threads.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1190)
		at org.apache.tomcat.util.threads.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:659)
		at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:63)
		at java.base/java.lang.Thread.run(Thread.java:1583)
	Caused by: java.sql.SQLSyntaxErrorException: Column 'EMPLOYEE_ID' not found.
		at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:112)
		at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:89)
		at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:81)
		at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:55)
		at com.mysql.cj.jdbc.result.ResultSetImpl.findColumn(ResultSetImpl.java:599)
		at com.mysql.cj.jdbc.result.ResultSetImpl.getInt(ResultSetImpl.java:872)
		at com.wavemaker.leavemanagement.repository.impl.EmployeeRepositoryImpl.getEmployeeIdByEmailId(EmployeeRepositoryImpl.java:70)
		... 26 more
29-Aug-2024 16:12:11.918 SEVERE [http-nio-8080-exec-7] org.apache.catalina.core.StandardWrapperValve.invoke Servlet.service() for servlet [com.wavemaker.leavemanagement.controller.LeavesController] in context with path [/leave_management] threw exception
	java.lang.RuntimeException: java.sql.SQLSyntaxErrorException: Column 'EMPLOYEE_ID' not found.
		at com.wavemaker.leavemanagement.repository.impl.EmployeeRepositoryImpl.getEmployeeIdByEmailId(EmployeeRepositoryImpl.java:82)
		at com.wavemaker.leavemanagement.service.LeavesService.getAllLeaves(LeavesService.java:18)
		at com.wavemaker.leavemanagement.controller.LeavesController.doGet(LeavesController.java:32)
		at jakarta.servlet.http.HttpServlet.service(HttpServlet.java:564)
		at jakarta.servlet.http.HttpServlet.service(HttpServlet.java:658)
		at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:195)
		at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:140)
		at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:51)
		at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:164)
		at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:140)
		at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:167)
		at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:90)
		at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:483)
		at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:115)
		at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:93)
		at org.apache.catalina.valves.AbstractAccessLogValve.invoke(AbstractAccessLogValve.java:663)
		at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:74)
		at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:344)
		at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:384)
		at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:63)
		at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:904)
		at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1741)
		at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:52)
		at org.apache.tomcat.util.threads.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1190)
		at org.apache.tomcat.util.threads.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:659)
		at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:63)
		at java.base/java.lang.Thread.run(Thread.java:1583)
	Caused by: java.sql.SQLSyntaxErrorException: Column 'EMPLOYEE_ID' not found.
		at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:112)
		at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:89)
		at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:81)
		at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:55)
		at com.mysql.cj.jdbc.result.ResultSetImpl.findColumn(ResultSetImpl.java:599)
		at com.mysql.cj.jdbc.result.ResultSetImpl.getInt(ResultSetImpl.java:872)
		at com.wavemaker.leavemanagement.repository.impl.EmployeeRepositoryImpl.getEmployeeIdByEmailId(EmployeeRepositoryImpl.java:70)
		... 26 more
29-Aug-2024 16:12:12.798 SEVERE [http-nio-8080-exec-8] org.apache.catalina.core.StandardWrapperValve.invoke Servlet.service() for servlet [com.wavemaker.leavemanagement.controller.TeamLeavesController] in context with path [/leave_management] threw exception
	java.lang.RuntimeException: java.sql.SQLSyntaxErrorException: Column 'EMPLOYEE_ID' not found.
		at com.wavemaker.leavemanagement.repository.impl.EmployeeRepositoryImpl.getEmployeeIdByEmailId(EmployeeRepositoryImpl.java:82)
		at com.wavemaker.leavemanagement.service.LeavesService.getAllLeaveRequests(LeavesService.java:28)
		at com.wavemaker.leavemanagement.controller.TeamLeavesController.doGet(TeamLeavesController.java:31)
		at jakarta.servlet.http.HttpServlet.service(HttpServlet.java:564)
		at jakarta.servlet.http.HttpServlet.service(HttpServlet.java:658)
		at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:195)
		at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:140)
		at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:51)
		at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:164)
		at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:140)
		at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:167)
		at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:90)
		at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:483)
		at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:115)
		at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:93)
		at org.apache.catalina.valves.AbstractAccessLogValve.invoke(AbstractAccessLogValve.java:663)
		at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:74)
		at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:344)
		at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:384)
		at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:63)
		at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:904)
		at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1741)
		at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:52)
		at org.apache.tomcat.util.threads.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1190)
		at org.apache.tomcat.util.threads.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:659)
		at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:63)
		at java.base/java.lang.Thread.run(Thread.java:1583)
	Caused by: java.sql.SQLSyntaxErrorException: Column 'EMPLOYEE_ID' not found.
		at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:112)
		at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:89)
		at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:81)
		at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:55)
		at com.mysql.cj.jdbc.result.ResultSetImpl.findColumn(ResultSetImpl.java:599)
		at com.mysql.cj.jdbc.result.ResultSetImpl.getInt(ResultSetImpl.java:872)
		at com.wavemaker.leavemanagement.repository.impl.EmployeeRepositoryImpl.getEmployeeIdByEmailId(EmployeeRepositoryImpl.java:70)
		... 26 more
29-Aug-2024 16:12:13.015 SEVERE [http-nio-8080-exec-9] org.apache.catalina.core.StandardWrapperValve.invoke Servlet.service() for servlet [com.wavemaker.leavemanagement.controller.TeamLeavesController] in context with path [/leave_management] threw exception
	java.lang.RuntimeException: java.sql.SQLSyntaxErrorException: Column 'EMPLOYEE_ID' not found.
		at com.wavemaker.leavemanagement.repository.impl.EmployeeRepositoryImpl.getEmployeeIdByEmailId(EmployeeRepositoryImpl.java:82)
		at com.wavemaker.leavemanagement.service.LeavesService.getAllLeaveRequests(LeavesService.java:28)
		at com.wavemaker.leavemanagement.controller.TeamLeavesController.doGet(TeamLeavesController.java:31)
		at jakarta.servlet.http.HttpServlet.service(HttpServlet.java:564)
		at jakarta.servlet.http.HttpServlet.service(HttpServlet.java:658)
		at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:195)
		at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:140)
		at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:51)
		at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:164)
		at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:140)
		at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:167)
		at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:90)
		at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:483)
		at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:115)
		at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:93)
		at org.apache.catalina.valves.AbstractAccessLogValve.invoke(AbstractAccessLogValve.java:663)
		at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:74)
		at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:344)
		at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:384)
		at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:63)
		at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:904)
		at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1741)
		at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:52)
		at org.apache.tomcat.util.threads.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1190)
		at org.apache.tomcat.util.threads.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:659)
		at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:63)
		at java.base/java.lang.Thread.run(Thread.java:1583)
	Caused by: java.sql.SQLSyntaxErrorException: Column 'EMPLOYEE_ID' not found.
		at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:112)
		at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:89)
		at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:81)
		at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:55)
		at com.mysql.cj.jdbc.result.ResultSetImpl.findColumn(ResultSetImpl.java:599)
		at com.mysql.cj.jdbc.result.ResultSetImpl.getInt(ResultSetImpl.java:872)
		at com.wavemaker.leavemanagement.repository.impl.EmployeeRepositoryImpl.getEmployeeIdByEmailId(EmployeeRepositoryImpl.java:70)
		... 26 more
29-Aug-2024 16:12:14.499 SEVERE [http-nio-8080-exec-10] org.apache.catalina.core.StandardWrapperValve.invoke Servlet.service() for servlet [com.wavemaker.leavemanagement.controller.LeavesController] in context with path [/leave_management] threw exception
	java.lang.RuntimeException: java.sql.SQLSyntaxErrorException: Column 'EMPLOYEE_ID' not found.
		at com.wavemaker.leavemanagement.repository.impl.EmployeeRepositoryImpl.getEmployeeIdByEmailId(EmployeeRepositoryImpl.java:82)
		at com.wavemaker.leavemanagement.service.LeavesService.getAllLeaves(LeavesService.java:18)
		at com.wavemaker.leavemanagement.controller.LeavesController.doGet(LeavesController.java:32)
		at jakarta.servlet.http.HttpServlet.service(HttpServlet.java:564)
		at jakarta.servlet.http.HttpServlet.service(HttpServlet.java:658)
		at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:195)
		at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:140)
		at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:51)
		at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:164)
		at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:140)
		at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:167)
		at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:90)
		at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:483)
		at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:115)
		at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:93)
		at org.apache.catalina.valves.AbstractAccessLogValve.invoke(AbstractAccessLogValve.java:663)
		at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:74)
		at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:344)
		at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:384)
		at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:63)
		at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:904)
		at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1741)
		at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:52)
		at org.apache.tomcat.util.threads.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1190)
		at org.apache.tomcat.util.threads.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:659)
		at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:63)
		at java.base/java.lang.Thread.run(Thread.java:1583)
	Caused by: java.sql.SQLSyntaxErrorException: Column 'EMPLOYEE_ID' not found.
		at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:112)
		at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:89)
		at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:81)
		at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:55)
		at com.mysql.cj.jdbc.result.ResultSetImpl.findColumn(ResultSetImpl.java:599)
		at com.mysql.cj.jdbc.result.ResultSetImpl.getInt(ResultSetImpl.java:872)
		at com.wavemaker.leavemanagement.repository.impl.EmployeeRepositoryImpl.getEmployeeIdByEmailId(EmployeeRepositoryImpl.java:70)
		... 26 more
29-Aug-2024 16:12:15.150 SEVERE [http-nio-8080-exec-1] org.apache.catalina.core.StandardWrapperValve.invoke Servlet.service() for servlet [com.wavemaker.leavemanagement.controller.TeamLeavesController] in context with path [/leave_management] threw exception
	java.lang.RuntimeException: java.sql.SQLSyntaxErrorException: Column 'EMPLOYEE_ID' not found.
		at com.wavemaker.leavemanagement.repository.impl.EmployeeRepositoryImpl.getEmployeeIdByEmailId(EmployeeRepositoryImpl.java:82)
		at com.wavemaker.leavemanagement.service.LeavesService.getAllLeaveRequests(LeavesService.java:28)
		at com.wavemaker.leavemanagement.controller.TeamLeavesController.doGet(TeamLeavesController.java:31)
		at jakarta.servlet.http.HttpServlet.service(HttpServlet.java:564)
		at jakarta.servlet.http.HttpServlet.service(HttpServlet.java:658)
		at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:195)
		at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:140)
		at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:51)
		at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:164)
		at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:140)
		at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:167)
		at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:90)
		at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:483)
		at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:115)
		at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:93)
		at org.apache.catalina.valves.AbstractAccessLogValve.invoke(AbstractAccessLogValve.java:663)
		at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:74)
		at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:344)
		at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:384)
		at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:63)
		at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:904)
		at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1741)
		at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:52)
		at org.apache.tomcat.util.threads.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1190)
		at org.apache.tomcat.util.threads.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:659)
		at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:63)
		at java.base/java.lang.Thread.run(Thread.java:1583)
	Caused by: java.sql.SQLSyntaxErrorException: Column 'EMPLOYEE_ID' not found.
		at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:112)
		at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:89)
		at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:81)
		at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:55)
		at com.mysql.cj.jdbc.result.ResultSetImpl.findColumn(ResultSetImpl.java:599)
		at com.mysql.cj.jdbc.result.ResultSetImpl.getInt(ResultSetImpl.java:872)
		at com.wavemaker.leavemanagement.repository.impl.EmployeeRepositoryImpl.getEmployeeIdByEmailId(EmployeeRepositoryImpl.java:70)
		... 26 more
29-Aug-2024 16:16:32.442 SEVERE [http-nio-8080-exec-10] org.apache.catalina.core.StandardWrapperValve.invoke Servlet.service() for servlet [com.wavemaker.leavemanagement.controller.LeavesController] in context with path [/leave_management] threw exception
	org.json.JSONException: JSONObject["employeeId"] not found.
		at org.json.JSONObject.get(JSONObject.java:596)
		at org.json.JSONObject.getInt(JSONObject.java:768)
		at com.wavemaker.leavemanagement.service.LeavesService.requestLeave(LeavesService.java:49)
		at com.wavemaker.leavemanagement.controller.LeavesController.doPost(LeavesController.java:65)
		at jakarta.servlet.http.HttpServlet.service(HttpServlet.java:590)
		at jakarta.servlet.http.HttpServlet.service(HttpServlet.java:658)
		at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:195)
		at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:140)
		at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:51)
		at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:164)
		at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:140)
		at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:167)
		at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:90)
		at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:483)
		at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:115)
		at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:93)
		at org.apache.catalina.valves.AbstractAccessLogValve.invoke(AbstractAccessLogValve.java:663)
		at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:74)
		at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:344)
		at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:384)
		at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:63)
		at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:904)
		at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1741)
		at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:52)
		at org.apache.tomcat.util.threads.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1190)
		at org.apache.tomcat.util.threads.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:659)
		at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:63)
		at java.base/java.lang.Thread.run(Thread.java:1583)
29-Aug-2024 16:36:41.239 SEVERE [http-nio-8080-exec-7] org.apache.catalina.core.StandardWrapperValve.invoke Servlet.service() for servlet [com.wavemaker.leavemanagement.controller.LeavesController] in context with path [/leave_management] threw exception
	org.json.JSONException: JSONObject["appliedOnDate"] not found.
		at org.json.JSONObject.get(JSONObject.java:596)
		at com.wavemaker.leavemanagement.service.LeavesService.requestLeave(LeavesService.java:65)
		at com.wavemaker.leavemanagement.controller.LeavesController.doPost(LeavesController.java:65)
		at jakarta.servlet.http.HttpServlet.service(HttpServlet.java:590)
		at jakarta.servlet.http.HttpServlet.service(HttpServlet.java:658)
		at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:195)
		at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:140)
		at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:51)
		at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:164)
		at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:140)
		at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:167)
		at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:90)
		at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:483)
		at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:115)
		at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:93)
		at org.apache.catalina.valves.AbstractAccessLogValve.invoke(AbstractAccessLogValve.java:663)
		at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:74)
		at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:344)
		at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:384)
		at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:63)
		at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:904)
		at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1741)
		at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:52)
		at org.apache.tomcat.util.threads.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1190)
		at org.apache.tomcat.util.threads.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:659)
		at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:63)
		at java.base/java.lang.Thread.run(Thread.java:1583)
29-Aug-2024 16:39:07.724 SEVERE [http-nio-8080-exec-8] org.apache.catalina.core.StandardWrapperValve.invoke Servlet.service() for servlet [com.wavemaker.leavemanagement.controller.LeavesController] in context with path [/leave_management] threw exception
	org.json.JSONException: JSONObject["appliedOnDate"] not found.
		at org.json.JSONObject.get(JSONObject.java:596)
		at com.wavemaker.leavemanagement.service.LeavesService.requestLeave(LeavesService.java:65)
		at com.wavemaker.leavemanagement.controller.LeavesController.doPost(LeavesController.java:65)
		at jakarta.servlet.http.HttpServlet.service(HttpServlet.java:590)
		at jakarta.servlet.http.HttpServlet.service(HttpServlet.java:658)
		at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:195)
		at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:140)
		at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:51)
		at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:164)
		at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:140)
		at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:167)
		at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:90)
		at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:483)
		at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:115)
		at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:93)
		at org.apache.catalina.valves.AbstractAccessLogValve.invoke(AbstractAccessLogValve.java:663)
		at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:74)
		at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:344)
		at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:384)
		at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:63)
		at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:904)
		at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1741)
		at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:52)
		at org.apache.tomcat.util.threads.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1190)
		at org.apache.tomcat.util.threads.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:659)
		at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:63)
		at java.base/java.lang.Thread.run(Thread.java:1583)
